core/app/components/pubs/PubEditor/PubEditor.tsx

```diff
@@ -44,8 +45,10 @@ const RelatedPubValueElement = ({
 				<p className="text-sm">
 					You are creating a Pub related to{" "}
 					<span className="font-semibold">{getPubTitle(relatedPub)}</span> through the{" "}
-					<span className="font-semibold">{fieldName}</span> pub field. Please enter a
-					value for this relationship.
+					<span className="font-semibold">{fieldName}</span> pub field.
+					{element.schemaName !== CoreSchemaType.Null && (
+						<span>Please enter a value for this relationship.</span>
+					)}
```

Comment:
I just noticed it was still saying to fill out a field if the schema was null (and then it wouldn't render an input to fill out)